fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanSummaryData.java

```diff
@@ -191,4 +222,31 @@ private static BigDecimal computeTotalRepaymentTransactionAmount(Collection<Loan
                 loanTransactions);
         return totalRepaymentTransaction.add(totalDownPaymentTransaction);
     }
+
+    private static BigDecimal computeTotalAccruedDueAmount(Collection<LoanSchedulePeriodData> periods) {
```

Comment:
Please rename this to reflect it is calculating the accrued interest amount: `computeTotalInterestAccruedDueAmount`